tests/framework/extensions/clusters/kubernetesversions/default.go

```diff
@@ -0,0 +1,71 @@
+package kubernetesversions
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/stretchr/testify/require"
+)
+
+func Default(t *testing.T, client *rancher.Client, provider string, kubernetesVersions []string) ([]string, error) {
```

Comment:
Sorry Josh one last nitpick, there is no need to pass the testing.T, you can use the "logrus" package to log. That is our standard. Use logrus for framework logging and testing.T for tests.

Reply:
will do! accommodating this request now :)

Reply:
testing.T is still needed to be passed in, as it is referenced in the `require.NoError(t, err)` lines within the switch cases, but have refactored `t.Log` => `logrus`.

Posting this comment here, to follow up w/ concern outlined above, but changes will be implemented in new PR mentioned below